src/odemis/gui/conf/file.py

```diff
@@ -245,6 +245,7 @@ def __init__(self):
         self.default.set("acquisition", "fn_count", "0")
         self.default.set("acquisition", "overlap", "0.06")
         self.default.set("acquisition", "autostig_period", "5")
+        self.default.set("acquisition", "autofocus_period", "5")
```

Comment:
Could you indicate the unit here in comment (and do the same for the autostig_period). It's in "number of RoA acquisitions", right?